### PR TITLE
improve top level docs

### PR DESCRIPTION
**Pull request by nslater**

### CODE REVIEW

File: DEVELOP.rst [hunk 1]
```diff
@@ -1,78 +1,91 @@
...
     http.cors.enabled: true
     http.cors.allow-origin: "*"
 
-Then visit the browser on ``http://localhost:9000/?base_uri=http://localhost:4200``.
+You can then use the ``base_uri`` parameter to specify the URL of your CrateDB
+node, like so::
+
+    http://localhost:9000/?base_uri=http://localhost:4200
+
+This  ``base_uri`` parameter intended for development use only.
```
Comment (chaudum) [inline DEVELOP.rst]: *is* intended ...

File: app/plugins/monitoring/README.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+and speed of ``SELECT``, ``INSERT``, ``UPDATE``, and
+``DELETE`` queries.
+
+Installation
+============
+
+For this plugin to work properly, make sure the ``/conf/plugins.json`` file
+contains the following::
+
+	{
```
Comment (chaudum) [inline app/plugins/monitoring/README.rst]: afaik rst requires indentation for code blocks.

also, please specify the code block language

`.. code-block:: json`

File: app/plugins/monitoring/README.rst [hunk 3]
```diff
@@ -0,0 +1,32 @@
+==================================
+CrateDB Admin UI Monitoring Plugin
+==================================
+
+The CrateDB admin UI monitoring plugin allows you to monitor the average number
+and speed of ``SELECT``, ``INSERT``, ``UPDATE``, and
+``DELETE`` queries.
```
Comment (chaudum) [inline app/plugins/monitoring/README.rst]: could go into previous line

**Comment by nslater** [comment]
@chaudum fixed everything, I think? except specifying the json syntax. but only bcus that feature isn't supported with rst and github previews afaik

**Pull request merged**